src/lavinmq/http/controller/queues.cr

```diff
@@ -41,6 +42,21 @@ module LavinMQ
           end
         end
 
+        get "/api/queues/:vhost/:name/unacked" do |context, params|
+          with_vhost(context, params) do |vhost|
+            refuse_unless_management(context, user(context), vhost)
+            unacked_messages = Hash(UInt64, UnackedMessage).new
+            @amqp_server.vhosts[vhost].queues[params["name"]].@consumers.each do |c|
+              c.@channel.@unacked.reject { |u| u.queue.name != params["name"] }.each do |u|
+                if consumer = u.consumer
+                  unacked_messages[u.tag] = UnackedMessage.new(u.tag, consumer.tag, u.delivered_at.to_s)
```

Comment:
consumer tag is only unique within a Channel, should we keep a reference to the Channel?

Reply:
`deliviered_at` is `Time.monotonic`, so the value doesn't mean anything in it self. What you probably want to do is something like `(RoughTime.monotonic - u.delivered_at).total_seconds`, to get how long it has been unacked for.

Reply:
> consumer tag is only unique within a Channel, should we keep a reference to the Channel?

Yes, that's probably a good idea! Added channel.name and a link to the channel on the unacked-page

Reply:
> deliviered_at is a Time::Span, so the value doesn't mean much in it self. What you probably want to do is something like (RoughTime.monotonic - u.delivered_at).total_seconds, to get how long it has been unacked for.

Yes 👍 Should we do total_seconds as float or just to_i? Not sure if we need to be more exact than whole seconds